client/src/utils/custom-hooks.js

```diff
@@ -0,0 +1,28 @@
+import { useState } from 'react';
+
+const useValue = (initialValue) => {
+  const [value, setValue] = useState(initialValue);
+  const handleValue = (event) => setValue(event.target.value);
+  const reset = () => setValue(initialValue);
+
+  return [value, handleValue, reset];
+};
+
+const useForm = (initialValue) => {
```

Comment:
@aeciorc  new custom hook i am using. although keep the previous one in case if someone wish to use it later